[2025-09-19T17:01:34.703+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-19T17:01:34.760+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mayo_pipeline.etl_labs manual__2025-09-19T17:01:33.407937+00:00 [queued]>
[2025-09-19T17:01:34.772+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mayo_pipeline.etl_labs manual__2025-09-19T17:01:33.407937+00:00 [queued]>
[2025-09-19T17:01:34.773+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-19T17:01:34.792+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): etl_labs> on 2025-09-19 17:01:33.407937+00:00
[2025-09-19T17:01:34.803+0000] {standard_task_runner.py:63} INFO - Started process 197 to run task
[2025-09-19T17:01:34.809+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mayo_pipeline', 'etl_labs', 'manual__2025-09-19T17:01:33.407937+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/mayo_pipeline.py', '--cfg-path', '/tmp/tmphr29_wbw']
[2025-09-19T17:01:34.813+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask etl_labs
[2025-09-19T17:01:34.903+0000] {task_command.py:426} INFO - Running <TaskInstance: mayo_pipeline.etl_labs manual__2025-09-19T17:01:33.407937+00:00 [running]> on host e345aeeb9fae
[2025-09-19T17:01:35.008+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='mayo_pipeline' AIRFLOW_CTX_TASK_ID='etl_labs' AIRFLOW_CTX_EXECUTION_DATE='2025-09-19T17:01:33.407937+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-19T17:01:33.407937+00:00'
[2025-09-19T17:01:35.009+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-19T17:01:35.062+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-19T17:01:35.064+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "/bin/bash -lc 'set -euo pipefail; cd /opt/project && python etl/flatten_mimic_labs.py'"]
[2025-09-19T17:01:35.076+0000] {subprocess.py:86} INFO - Output:
[2025-09-19T17:01:38.200+0000] {subprocess.py:93} INFO - Wrote 95,246 rows to /opt/project/data/interim/labs.csv
[2025-09-19T17:01:38.266+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-19T17:01:38.267+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-19T17:01:38.408+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mayo_pipeline, task_id=etl_labs, run_id=manual__2025-09-19T17:01:33.407937+00:00, execution_date=20250919T170133, start_date=20250919T170134, end_date=20250919T170138
[2025-09-19T17:01:38.440+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-19T17:01:38.461+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-19T17:01:38.464+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
